core/story-api/src/main/java/org/lgna/story/event/MouseClickEvent.java

```diff
@@ -55,20 +55,4 @@ public abstract class MouseClickEvent extends AbstractEvent {
   public MouseClickEvent(MouseClickEventImp e) {
     this.implementation = e;
   }
-
```

Comment:
These removals leave unused imports here and in the corresponding Imp class that trigger CheckStyle.
(I should figure out how to automate the fixing rather than the complaining. Maybe move this to a GitHub action?)